### DISCUSSION TITLE
how to use laragon to create mobile app?

### DISCUSSION THREAD

**Discussion by mohamedirfan02** [leokhoa/laragon#955] [Q&A]
i have started to build mobile app but facing the error ;
Error: SocketException: Failed host lookup: 'http://localhost' (OS Error: No such host is known.
, errno = 11001)

**Comment by ppfoong** [comment]
You need to find out the IP address of the computer where you run Laragon. It is displayed on top of Laragon window.

Use that IP address for your URL.

Also, if you have SSL enabled for Apache, you should use https instead of HTTP
